### DISCUSSION TITLE
Move required binaries between caches

### DISCUSSION THREAD

**Discussion by iskiselev** [microsoft/vcpkg#33264] [Q&A]
As I understand, currently if VCPKG builds binary and binary cache is enabled, it would be pushed into all write-enabled caches.
If vcpkg find the required binary in one of the caches, build would be skipped and used binary would not be re-published to other caches.
Let's say I have local and http cache configure, but on initial build local cache is empty, but required binaries are in http cache. Is it possible to republish all binaries downloaded from http cache to local files cache, to speed up future builds?
I can't just copy all filed from http cache - there will be much fore versions cached that are required locally.

**Comment by autoantwort** [comment]
So more or less the same as in https://github.com/microsoft/vcpkg/discussions/31500
